content/interfaces/sql/dolt-system-tables.md

```diff
@@ -76,11 +76,91 @@ INSERT INTO dolt_branches (name, hash)
 VALUES ("my branch name", @@mydb_head);
 ```
 
+### `dolt_commit_diff_$TABLENAME`
+
+### Description 
+
+For every user table named `$TABLENAME`, there is a queryable system table names `dolt_commit_diff_$TABLENAME` which can be queried to see how a table has changed between two commits (regardless of which branch the user is currently using). Each row in the result set represents a row that has changed between two commits. 
```

Comment:
"system table _named_"

Reply:
Need to explain the `WHERE to_commit = '...' AND from_commit = '...';` requirement